language/evm/hardhat-examples/contracts/FortyTwo.move

```diff
@@ -12,10 +12,10 @@ module 0x1::FortyTwo {
     }
 
     // TODO: move-to-yul does not support literal string.
-    #[callable(sig=b"forty_two_as_string() returns (string)"), pure]
-    public fun forty_two_as_string(): vector<u8> {
-         b"forty two"
-    }
+    // #[callable(sig=b"forty_two_as_string() returns (string)"), pure]
```

Comment:
does this still cause an error?

Reply:
Reenabled it already in a recent push. Sorry for the confusion